### PR TITLE
Add an endpoint to check the version

### PR DESCRIPTION
**Pull request by shifucun**
image

### CODE REVIEW

File: server/templates/version.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  <link rel="stylesheet" href={{url_for('static', filename='css/static.min.css', t=config['GAE_VERSION'])}} >
+{% endblock %}
+
+{% block content %}
+  <div style="margin-left: 20px;">
+    <table>
+      <tbody>
+        <tr>
+          <td width="200px">
+            website commit
```
Comment (pradh) [inline server/templates/version.html:28]: Website Commit:

File: server/templates/version.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+          </td>
+          <td>
+            <a href="https://github.com/datacommonsorg/website/commit/{{website_hash}}">{{website_hash}}</a>
+          </td>
+        </tr>
+        <tr>
+          <td width="200px">
+            mixer commit
```
Comment (pradh) [inline server/templates/version.html:36]: Mixer Commit:

File: server/templates/version.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+          </td>
+          <td>
+            <a href="https://github.com/datacommonsorg/mixer/commit/{{mixer_hash}}">{{mixer_hash}}</a>
+          </td>
+        </tr>
+        <tr>
+          <td width="200px">
+            bigtable
```
Comment (pradh) [inline server/templates/version.html:44]: Bigtable Table:

File: server/templates/version.html [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+          </td>
+          <td>
+            {{bigtable}}
+          </td>
+        </tr>
+        <tr>
+          <td width="200px">
+            bigquery
```
Comment (pradh) [inline server/templates/version.html:52]: Bigquery Dataset:

**Review by pradh — APPROVED** [review]
Nice!

File: server/templates/version.html (hunk 1 above)
Comment (shifucun) [reply server/templates/version.html]: Updated and below

**Review by shifucun — COMMENTED** [review]
thanks for review!

**Review by beets — APPROVED** [review]
nice!

**Comment by beets** [comment]
any concerns with including git in the deployment?

**Comment by shifucun** [comment]
> any concerns with including git in the deployment?

The git hash is open information so I fell it's okay to expose it. The hash version is written to a config file, it has small impact on the deployed artifacts.

**Pull request merged**